jasmine/dependencies.bzl

```diff
@@ -14,36 +13,28 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 # changes in this function should be marked as BREAKING in the commit message
 # and released only in semver majors.
 def rules_jasmine_dependencies():
-    # The minimal version of bazel_skylib we require
-    maybe(
-        http_archive,
+    http_archive(
         name = "bazel_skylib",
-        sha256 = "c6966ec828da198c5d9adbaa94c05e3a1c7f21bd012a0b29ba8ddbccb2c93b0d",
-        urls = [
-            "https://github.com/bazelbuild/bazel-skylib/releases/download/1.1.1/bazel-skylib-1.1.1.tar.gz",
-            "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.1.1/bazel-skylib-1.1.1.tar.gz",
-        ],
+        sha256 = "74d544d96f4a5bb630d465ca8bbcfe231e3594e5aae57e1edbf17a6eb3ca2506",
+        urls = ["https://github.com/bazelbuild/bazel-skylib/releases/download/1.3.0/bazel-skylib-1.3.0.tar.gz"],
     )
-    
-    maybe(
-        http_archive,
+
+    http_archive(
         name = "aspect_bazel_lib",
-        sha256 = "79381b0975ba7d2d5653239e5bab12cf54d89b10217fe771b8edd95047a2e44b",
-        strip_prefix = "bazel-lib-1.12.1",
-        url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v1.12.1.tar.gz",
+        sha256 = "eae670935704ce5f9d050b2c23d426b4ae453458830eebdaac1f11a6a9da150b",
+        strip_prefix = "bazel-lib-1.15.0",
+        url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v1.15.0.tar.gz",
     )
 
-    maybe(
-        http_archive,
+    http_archive(
         name = "aspect_rules_js",
-        sha256 = "db9f446752fe4100320cf8487e8fd476b9af0adf6b99b601bcfd70b289bb0598",
-        strip_prefix = "rules_js-1.1.2",
-        url = "https://github.com/aspect-build/rules_js/archive/refs/tags/v1.1.2.tar.gz",
+        sha256 = "57b1e50da38c3b58c4fb970c3f665bd2b9705158b38a3fe9d38d7da267925c11",
+        strip_prefix = "rules_js-a47780ac9253344cc32b28c22c6da570fc05f190",
+        url = "https://github.com/aspect-build/rules_js/archive/a47780ac9253344cc32b28c22c6da570fc05f190.tar.gz",
```

Comment:
Lets wait for the release before merging though?

Reply:
Yup